src/API/OnboardingTasks.php

```diff
@@ -114,6 +114,12 @@ public function register_routes() {
 			$this->namespace,
 			'/' . $this->rest_base,
 			array(
+				'args'   => array(
+					'only_visible' => array(
```

Comment:
What do you think about `filter_visible` for this name?

Reply:
I'm open, although `filter_visible` to me is a little ambiguous. It could mean filter _out_ visible tasks, or filter for _only_ visible tasks. I'm up for changing it, but `only_visible` feels clearer to me between the two.